src/component/contents/DraftEditorTextNode.react.js

```diff
@@ -95,8 +95,10 @@ class DraftEditorTextNode extends React.Component {
     // an extra span with a comment in it, Gecko's code doesn't destroy our content.
     return (
       <span key={this._forceFlag ? 'A' : 'B'} data-text="true">
-        <span style={displayNone} dangerouslySetInnerHTML={{__html: '<!-- gecko patch -->' }}></span>
-        {this.props.children}
+        <span className='mozilla-fix' style={mozillaFix} dangerouslySetInnerHTML={{__html: '<!-- mozilla fix -->' }}></span>
+        <span data-text="true">
```

Comment:
Do we need to specify that this is also a `data-text` node? This looks like it will affect the logic in https://github.com/textioHQ/frontend/blob/topic-helens/app/es/components/editor/tooltipHandler.js#L91